tests/sentry/grouping/fingerprint_inputs/fingerprint-exception-type-and-sdk-mismatch.json

```diff
@@ -0,0 +1,36 @@
+{
+  "_fingerprinting_rules": [
+    {
+      "matchers": [
+        ["sdk", "sentry.javascript.nextjs"],
+        ["type", "ChunkLoadError"]
+      ],
+      "fingerprint": ["chunkloaderror"]
+    }
+  ],
+  "fingerprint": ["my-route", "{{ default }}"],
+  "exception": {
+    "values": [
+      {
+        "stacktrace": {
+          "frames": [
+            {
+              "function": "main",
+              "abs_path": "foo/bar.tsx",
+              "module": "foo.bar",
+              "filename": "bar.tsx",
+              "lineno": 13,
+              "in_app": false
+            }
+          ]
+        },
+        "type": "ChunkLoadError",
+        "value": "ChunkLoadError something something..."
+      }
+    ]
+  },
+  "platform": "javascript",
+  "sdk": {
+    "name": "sentry.javascript.ember"
```

Comment:
It will mismatch, therefore, it will use the system grouping
<img width="318" alt="image" src="https://github.com/getsentry/sentry/assets/44410/cdb329d5-7907-477b-944b-59d94eb964ee">